elegant/clean_timepoint_data.py

```diff
@@ -154,7 +154,7 @@ def remove_excluded_positions(experiment_root,dry_run=False):
                 with (experiment_root /  'experiment_metadata.json').open('r') as md_file:
                     expt_md = json.load(md_file)
                 del expt_md['positions'][position]
-                datafile.json_encode_atomic_legible_to_file(expt_md, md_file)
+                datafile.json_encode_atomic_legible_to_file(expt_md, experiment_root /  'experiment_metadata.json')
```

Comment:
This is a good example of why variable names should be consistent across functions: everywhere else, `md_file` is a pathlib.Path instance of the location of the metadata file. Here it is the open file pointer (`md_fp` elsewhere). Hence the copy-paste bug.

I sugest doing the same here, rather than constructing the same path twice. Plus then the variables will be more consistent :)